#! /bin/sh /usr/share/dpatch/dpatch-run
## 18_fig2ps2tex_bashisms.dpatch by Chris Lamb <chris@chris-lamb.co.uk>

## DP: Fix Bashisms in fig2ps2tex /bin/sh script

@DPATCH@
diff -urNad transfig-3.2.5-rel.orig/fig2dev/fig2ps2tex.sh.script transfig-3.2.5-rel/fig2dev/fig2ps2tex.sh.script
--- transfig-3.2.5-rel.orig/fig2dev/fig2ps2tex.sh.script	2008-05-24 16:04:20.000000000 +0100
+++ transfig-3.2.5-rel/fig2dev/fig2ps2tex.sh.script	2008-05-24 16:27:35.000000000 +0100
@@ -14,11 +14,11 @@
 xsp=`echo "scale=3; ( $bbox4 - $bbox2 ) / 72" | bc`
 ysp=`echo "scale=3; ( $bbox5 - $bbox3 ) / 72" | bc`
 
-echo "\makebox[${xsp}in][l]{"
-echo "  \vbox to ${ysp}in{"
-echo "    \vfill"
-echo "    \special{psfile=$1}"
-echo "  }"
-echo "  \vspace{-\baselineskip}"
-echo "}"
+printf "\makebox[${xsp}in][l]{\n"
+printf "  \\\vbox to ${ysp}in{\n"
+printf "    \\\vfill\n"
+printf "    \special{psfile=$1}\n"
+printf "  }\n"
+printf "  \\\vspace{-\\\baselineskip}\n"
+printf "}\n"
 
